dashboard/django/stats/templates/stats/brokers.html

```diff
@@ -0,0 +1,87 @@
+<!--
+
+    Copyright 2016 Yahoo Inc.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+{% extends "stats/base.html" %}
+
+{% load humanize %}
+{% load table %}
+{% load stats_extras %}
+
+{% block title %}Brokers{% endblock %}
+
+{% block content %}
+
+<div class="module filtered" id="changelist">
+<div id="changelist-filter">
+    <h2>Clusters</h2>
+<ul>
+   <li {% if not selectedCluster %}class="selected"{% endif %}>
```

Comment:
here it seems, it doesn't show broker with 0 bundles. broker has to own at least one bundle to appear here. should we show regardless number of bundle.